doc/releases/changelog-dev.md

```diff
@@ -211,6 +211,9 @@
 
 <h3>Bug fixes</h3>
 
+* Resolve a bug where `mitigate_with_zne` is not working with the transform `measurements_from_sample`.
```

Comment:
Could you add in what scenario the user would encounter this combination? (since measurements from sample is not something they would use directly)

Reply:
Using a device that has only {Counts, Samples} supported for example qrack with the ZNE transform. Not sure how to include that?

Reply:
Updated text, let me know what you think.